src/Ldap.rb

```diff
@@ -436,6 +436,7 @@ def ReadLdapHosts
     # Reads LDAP settings from the SCR
     # @return success
     def Read
+      return true unless @read_settings
```

Comment:
please document this change that it reads only if it is not read before and also that you can force read by calling
```
Ldap.SetReadSettings = nil
Ldap.Read
```